src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,385 @@
+* **[Reliability](/docs/platform/architecture/fault-tolerance):** 99.999% uptime SLA, with automatic failover and seamless re-connection.
+* **[Availability](/docs/platform/architecture/edge-network):** Global edge infrastructure ensures users connect to the closest point for optimal experience.
+
+![Ably Architecture Overview Diagram](../../../../images/content/diagrams/architecture-overview.png)
+
+Delivering chat messages in realtime is key to an engaging experience. Ably's [serverless architecture](/docs/platform/architecture) eliminates the need for you to manage websocket servers. It automatically scales to handle millions of concurrent connections without provisioning or maintenance. Ably also handles all of the edge-cases around delivery, failover and scaling.
+
+Despite the challenges of delivering these guarantees, Ably is designed to keep costs predictable.
+You can choose between MAU-based or per-minute pricing, depending on your needs.
+Both pricing models are designed to be cost-effective and scalable.
+Features such as efficient connection management are available to help you reduce costs.
+
+## Architecting your task-oriented chat: UX first, scale always
+
+The most important decision when developing task-oriented chat is the experience you want users to have within their specific context. This will determine how chat integrates with your task flow and ultimately how effectively it enhances the user experience.
+
+**What are task-oriented chats?** Any chat that serves as a companion to a specific activity or workflow: resolving a support ticket, coordinating a delivery, discussing a chess game, collaborating on a project, or facilitating a marketplace transaction. These chats are typically short-lived (minutes to days) and highly contextual. They can be 1:1 or small group conversations.
+
+For large-scale chat scenarios, see our [building livestream chat guide](/docs/guides/chat/build-livestream).
+
+**Task-oriented room architecture:** The optimal approach is one chat room per task, where all participants join to communicate within that specific context. Rooms are:
+
+* **Automatically scaled** - Ably handles any number of concurrent rooms without provisioning
+* **History-enabled** - messages stored for 30 days by default (configurable up to a year)
+* **Kept under control** - [capabilites](/docs/chat/setup#authentication) let you control who can join and the level of access to features and actions
+
+Each room includes all task participants: customers and agents in support, buyers and sellers in marketplaces, players in games, team members in projects. Participants can join and leave as the task evolves. Access to message history and other features can be granted to all or select participants via capabilities.
+
+### Pricing flexibility
+
+Depending on the specifics of your platform, you can choose between per-minute or per-MAU pricing.
```

Comment:
Not strictly per-minute, it's "consumption-based". Whereas channels and connections are per-minute, messages are not

Reply:
I initially wrote "consumption-based" but then I've changed to use the same wording from the pricing page ("pay per minute" or "pay per MAU"). Happy with either. But you asked a good question on whether this section should even exist (as it will have to be kept updated with any pricing model changes)